.github/workflows/ci.yml

```diff
@@ -814,6 +814,8 @@ jobs:
     needs: [detect-changes]
     uses: ./.github/workflows/zeebe-ci.yml
     secrets: inherit
+    with:
+      stable-branch: ${{ detect-changes.outputs.stable-branch-changes == 'true' }}
```

Comment:
Needs the `needs.` in the beginning: `needs.detect-changes.outputs.stable-branch-changes == 'true'`